### PR TITLE
[MOS-273] Update Harness to detect Mudita Bell devices

### PR DESCRIPTION
**Pull request by niepiekm**
Added detection of Mudita Harmony

### CODE REVIEW

**Review by mjan-dev — APPROVED** [review]

File: test/pytest/conftest.py [hunk 1]
```diff
@@ -121,7 +121,7 @@ def harnesses():
     '''
     Automatically init at least two Pure phones
     '''
-    found_pures = serial.find_Pures()
+    found_pures = serial.find_Devices()
```
Comment (SP2FET) [inline test/pytest/conftest.py:124]: so now it should be renamed :D
Also - wouldn't adding a Harmony on the devices lists break pure-specific tests?

Comment (niepiekm) [reply test/pytest/conftest.py:124]: The method is renamed to indicate a new purpose - it now looks not only for Pure :)

If we run Pure-specific tests on Harmony they will fail, obviously. 
Making harness work with Harmony is not necessary for development, I just noticed the lack of it while testing CP-1103. This change can be postponed.

Comment (SP2FET) [reply test/pytest/conftest.py:124]: i meant the variable's name :D

Comment (niepiekm) [reply test/pytest/conftest.py]: Fixed :)

**Review by SP2FET — APPROVED** [review]

**Pull request merged**